src/repository/screens/repository.screen.js

```diff
@@ -158,168 +162,212 @@ class Repository extends Component {
-            />
-          </SectionList>
+            {!isPendingContributors &&
+              <MembersList
+                title="CONTRIBUTORS"
+                members={contributors}
+                navigation={navigation}
+              />}
 
-          <SectionList
-            loading={isPendingIssues}
-            title={
-              <RepositorySectionTitle
-                text="ISSUES"
-                openCount={openIssues.length}
-                closedCount={closedIssues.length}
+            <SectionList title="SOURCE">
+              <ListItem
+                title="README"
+                leftIcon={{
+                  name: 'book',
+                  color: colors.grey,
+                  type: 'octicon',
+                }}
+                titleStyle={styles.listTitle}
+                onPress={() =>
+                  navigation.navigate('ReadMe', {
+                    repository,
+                  })}
+                underlayColor={colors.greyLight}
               />
-            }
-            noItems={openIssues.length === 0}
-            noItemsMessage={
-              pureIssues.length === 0 ? 'No issues' : 'No open issues'
-            }
-            showButton={pureIssues.length > 0}
-            buttonTitle="View All"
-            buttonAction={() =>
-              navigation.navigate('IssueList', {
-                type: 'issue',
-                issues: pureIssues,
-              })}
-          >
-            {openIssues
-              .slice(0, 3)
-              .map(item =>
-                <IssueListItem
-                  key={item.id}
-                  type="issue"
-                  issue={item}
-                  navigation={navigation}
+              <ListItem
+                title="View Code"
+                titleStyle={styles.listTitle}
+                leftIcon={{
+                  name: 'code',
+                  color: colors.grey,
+                  type: 'octicon',
+                }}
+                onPress={() =>
+                  navigation.navigate('RepositoryCodeList', {
+                    topLevel: true,
+                  })}
+                underlayColor={colors.greyLight}
+              />
+            </SectionList>
+
+            <SectionList
+              loading={isPendingIssues}
+              title="ISSUES"
+              noItems={
+                pureIssues.filter(issue => issue.state === 'open').length === 0
+              }
+              noItemsMessage={
+                pureIssues.length === 0 ? 'No issues' : 'No open issues'
+              }
+              showButton={pureIssues.length > 0}
+              buttonTitle="View All"
+              buttonAction={() =>
+                navigation.navigate('IssueList', {
+                  type: 'issue',
+                  issues: pureIssues,
+                })}
+            >
+              {pureIssues
+                .filter(issue => issue.state === 'open')
+                .slice(0, 3)
```

Comment:
I think you can create a constant telling what does this `3` means, it will make things clearer for everyone.